seed/seed.json

```diff
@@ -942,6 +942,44 @@
                 "channel": ["RELEASE", "BETA", "NIGHTLY"],
                 "platform": ["WINDOWS", "MAC", "LINUX"]
             }
+        },
+        {
+            "name": "OperationalPatternsCollectionStudyBeta",
```

Comment:
Beta or Nightly?
```"channel": ["NIGHTLY"],```

Reply:
Good point. We want to run it on Nightly first. I'm just going to remove the channel from the study title so we don't have to rename it each time